webrtc/videoroom/assets/src/membraneWebRTC.ts

```diff
@@ -200,6 +220,14 @@ export class MembraneWebRTC {
     }
   };
 
+  public toggleVideo = () => {
+    this.channel.push("toggleVideo", {});
```

Comment:
`toggledVideo`?

So server will receive info `toggledVideo, peer_pid` and send messages `participantToggledVideo, mid`